### PR TITLE
Relax face tracker

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5a556a06d29be0dde4c0d0ae8e1d26fe16e8c8a9 available at: https://gist.github.com/442322c43c19e82bcc75<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.dmg

**Comment by Atlante45** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5a556a06d29be0dde4c0d0ae8e1d26fe16e8c8a9 available at: https://gist.github.com/35fd086413eebd8ce0f0<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.dmg

File: interface/src/devices/FaceTracker.cpp [hunk 1]
```diff
@@ -11,7 +11,36 @@
...
+}
+
+void FaceTracker::update(float deltaTime) {
+    static const float RELAXATION_TIME = 0.4; // sec
+    
+    if (isTracking()) {
+        if (_relaxationStatus == 1.0f) {
+            return;
+        }
+        _relaxationStatus += deltaTime / RELAXATION_TIME;
```
Comment (PhilipRosedale) [inline interface/src/devices/FaceTracker.cpp]: I thought that something didn't look quite right!  :)

you want expontential:  

_relaxationStatus *= (1.0 - deltaTime / RELAXATION_TIME); 

or 

_relaxationStatus *= (1.0 + deltaTime / RELAXATION_TIME); 

and then check for when _relaxationStatus is within EPSILON of 1.0 or 0.0 and set it to the value exactly.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9849efe0130082673622349ffc45559e68beafb4 available at: https://gist.github.com/fe849b96064fda6107d5<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8615bb7cff3a39a37cd50301977e35c58b4bd6d0 available at: https://gist.github.com/e2b9e593b1be4ec8eda5<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: db90fcdb6220a7a377cd816d765631230dcbc0a5 available at: https://gist.github.com/223fdd3308f4ddb77be1<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4348.dmg

**Pull request merged**